org.sixtyninestudios.kublit.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 69 Studios: A Kallex Company -->
+<component type="desktop-application">
+  <id>org.sixtyninestudios.kublit</id>
```

Comment:
`sixtyninestudios.org` doesn't seem to exist.

Reply:
We are new to flatpak, check the yml for the id to match

Reply:
Not sure what you are saying here. But from the documentation you acknowledged having read:
https://github.com/flathub/flathub/wiki/App-Requirements#application-id
> The Application ID should be a real URL of a domain that the app author has control over or where their app is hosted.

Reply:
What should the id be then? I don't understand how it's a bad id?

Reply:
https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
Yeah, it should be a valid id

69 studios is a organization that make games

Sixtyninestudios is the name of the organization, the numbers are words as I don't think actual numbers are allowed in id's, and then there's the project name

Can you point out what's wrong with it?

Reply:
> Sixtyninestudios is the name of the organization, 

AND? We are talking about domain name, and I linked to the section of the documentation that discuss using app-ids based on code hosting, which is perfectly fine.

I already pointed out what was wrong above.

Reply:
Our website domain is 69-studios.co.uk/

Reply:
`uk.co.69_studios.kublit` would be your app-id.

Reply:
Thx